ui/src/index.scss

```diff
@@ -3,11 +3,16 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&display=swap');
 
+@import 'overrides/overrides.scss';
 @import 'colors.scss';
 @import 'icons/Icons.scss';
 
+@import 'components/App.scss';
+
+@include overrides;
```

Comment:
I stuck most (though not all) of the styles that were simply changing default semantic UI classes into `overrides`